edi_endpoint_oca/data/server_action.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo noupdate="1">
+    <record id="server_action_registry_sync" model="ir.actions.server">
+        <field name="name">Sync registry</field>
+        <field name="type">ir.actions.server</field>
+        <field name="model_id" ref="edi_endpoint_oca.model_edi_endpoint" />
+        <field name="binding_model_id" ref="edi_endpoint_oca.model_edi_endpoint" />
+        <field name="state">code</field>
+        <field name="code">
+records.filtered(lambda x: not x.registry_sync).write({"registry_sync": True})
```

Comment:
Same remark as for https://github.com/OCA/web-api/pull/78 :wink:

Reply:
@simahawk done! also included https://github.com/OCA/edi/pull/1110